libraries/libraries/settings/base.py

```diff
@@ -143,6 +139,6 @@
 
 # Base URL to use when referring to full URLs within the Wagtail admin backend -
 # e.g. in notification emails. Don't include '/admin' or a trailing slash
-BASE_URL = 'http://example.com'
+BASE_URL = 'http://libraries-dev.cca.edu'
```

Comment:
Can you put this in dev.py instead? I guess "BASE_URL" should probably be http://localhost in base, libraries-dev.cca.edu in dev, and libraries.cca.edu in production.

I don't actually know how Django knows which file to use…I assume either environmental variables or some parameter passed to the app on startup.